cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/monitor/RuntimeMonitor.java

```diff
@@ -128,8 +118,8 @@ protected void run() {
             throw new ServiceUnavailableException(response.getResponseBodyAsString());
           }
 
-          Map<String, List<MonitorMessage>> monitorResponses =
-            GSON.fromJson(response.getResponseBodyAsString(StandardCharsets.UTF_8), MAP_STRING_MESSAGE_TYPE);
+          List<TopicMessage> monitorResponses =
+            GSON.fromJson(response.getResponseBodyAsString(StandardCharsets.UTF_8), LIST_MESSAGE_TYPE);
```

Comment:
Is this correct? It seems like the response is of type `Map<String, List<MonitorMessage>>`?